lib/data/wrapper.js

```diff
@@ -696,6 +715,19 @@ const data = {
             });
         }
 
+        if (locationTypeMatchGCP && dataLocator.lenegth === 1) {
```

Comment:
Typo: `lenegth` => `length`

Reply:
will fix this.

Reply:
Those `e` chars don't stay in one place! 😸